docs/gestor-dependencias.md

```diff
@@ -0,0 +1,8 @@
+# Gestor de dependencias
+A la hora de elegir un gestor de dependencias, hay ciertos aspectos que debemos tener en cuenta. Estos son eficiencia, mantenimiento y, por supuesto, que siga los estándares que hay en el lenguaje, cosa que cumplen todos los gestores que vamos a ver.
+- [npm](https://www.npmjs.com/) no destaca por su eficiencia, aunque en la última versión ha mejorado bastante. Está en constante desarrollo y mejora. 
+- [yarn](https://yarnpkg.com/) es más rápido que npm debido que usa un sitema de caché para la instalación de las dependencias además de realizar la descarga de las mismas en paralelo. Es un proyecto consolidado pero lleva tiempo sin actualizarse.
+- [pnpm](https://pnpm.io/) usa un sistema de caché parecido al de Yarn. Usa un sistema de almacenamiento compartido entre los diferentes proyectos, evitando duplicidades y reduciendo la carga a la hora de instalar las dependencias. Es muy rápido, llegando incluso a doblar a las otras alternativas que estamos comentando, y útil en proyectos grandes. Está creciendo por lo que está en continuo desarrollo y mantenimiento. 
+
+## Elección
+Aunque el proyecto en el que vamos a trabajar no es muy grande y no le vamos a sacar el máximo partido, elijo pnpm como gestor de dependencias por ser el más rápido de los nombrados, además de que es el que más está creciendo y mejorando.
```

Comment:
Por favor, Jaime, haz un poco de caso a lo que te digo. Arriba listas 3 criterios: eficiencia, mantenimiento, y que siga los estándares. OK, los estándares los siguen los tres. Aquí lo eliges "por ser el más rápido". Estaría bien que pusieras un enlace a algún sitio donde afirme eso, pero en todo caso, ¿qué pasa con el mantenimiento? ¿"Creciendo y  mejorando" == "mantenimiento"?
Tenéis total libertad para elegir los criterios (dentro de que efectivamente sean objetivos y adecuados para la herramienta), pero por favor, dentro de eso, tratad de ser coherentes con lo que hacéis.